packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -45,24 +45,47 @@ import {
 import { SegmentGroupCollection } from "./segmentGroupCollection";
 import { PropertiesManager, PropertiesRollback } from "./segmentPropertiesManager";
 
+/**
+ * Common properties for a node in a merge tree.
```

Comment:
Is there a concrete type we can link to from here for "merge tree"? Might enhance the resulting docs a bit.

Reply:
It's being removed from public export so that the API surface can be changed without it being breaking.

These docs are all primarily going to be useful for merge-tree developers, not Fluid consumers anyway. So referencing the type is likely redundant (the package is called merge-tree; people probably got to these types from looking first at merge-tree anyway)